### ISSUE TITLE
SpaceInsideParens style checker does not recognize space after parens in some conditions

### ISSUE THREAD

**Issue by joshchu** [rubocop/rubocop#1326]
Test case to reproduce:

```
  it 'reproduces issue that SpaceInsideParens does not catch space on LHS parens' do
    inspect_source(cop, ['test = ( 4 * 60 * 60)'])
    expect(cop.messages).not_to be_empty
  end
```

I suspect there's some sort of parse issue with SurroundingSpace but haven't had a chance to investigate/debug further
